bananas_api/user/developer.py

```diff
@@ -13,17 +13,29 @@ class User(BaseUser):
     routes = web.RouteTableDef()
 
     def get_authorize_page(self):
+        # assume developer auth happens on http and not https. If it does happen to be on https,
+        #  the developer should be smart enough to know how to fix the returned url by themselves.
+        # While for a browser session, we could suffice with a relative url, because this url is shown in the
+        #  cli for copy-pasting, we need to include the host name as well.
```

Comment:
such a long comment for something you do not do :P Or am I missing something here?

Reply:
I did it in a previous iteration, but forgot to remove this comment, I should get on that 😅